### ISSUE TITLE
Endpoint /transactions.all test is randomly failing

### ISSUE THREAD

**Issue by sirn** [omisego/ewallet#147]
# Subject of the issue

```
  1) test /transactions.all returns all the transactions (EWalletAPI.V1.TransactionControllerTest)
     test/ewallet_api/v1/controllers/transaction_controller_test.exs:42
     Assertion with == failed
     code:  assert Enum.map(response["data"]["data"], fn t -> t["id"] end) == [meta.transfer_1().id(), meta.transfer_2().id(), meta.transfer_3().id(), meta.transfer_4().id(), meta.transfer_5().id(), meta.transfer_6().id(), meta.transfer_7().id(), meta.transfer_8().id()]
     left:  ["a6c65166-7694-476e-b031-5879f97ce60e",
             "3b38eb21-d483-4f58-ba67-68e8e2f04ad1",
             "2b23faf0-c724-4ded-9764-438fba02eb69",
             "2f2816d2-e60f-4a57-a8be-85e8bbf7e90c",
             "c18165f0-18e7-405c-b825-99a2497c0774",
             "b83cc2e9-8285-4970-a4dd-f52c2e96fca9",
             "5ab273af-861e-4b26-8f36-8d4fbb27c210",
             "6129e74e-86cc-4fb2-ae9b-d94b350b7000"]
     right: ["2b23faf0-c724-4ded-9764-438fba02eb69",
             "2f2816d2-e60f-4a57-a8be-85e8bbf7e90c",
             "c18165f0-18e7-405c-b825-99a2497c0774",
             "b83cc2e9-8285-4970-a4dd-f52c2e96fca9",
             "5ab273af-861e-4b26-8f36-8d4fbb27c210",
             "6129e74e-86cc-4fb2-ae9b-d94b350b7000",
             "a6c65166-7694-476e-b031-5879f97ce60e",
             "3b38eb21-d483-4f58-ba67-68e8e2f04ad1"]
     stacktrace:
       test/ewallet_api/v1/controllers/transaction_controller_test.exs:48: (test)
```

# Your environment

Jenkins, CI

# Steps to reproduce

Run the test few times.

# Expected behavior

Not randomly failing

# Actual behavior

Randomly failing

**Comment by sirn** [comment]
Ping :(

**Comment by unnawut** [comment]
Fixed with #154
